test/functional/functional_test_utils.py

```diff
@@ -641,3 +645,13 @@ def client_cycle_batch_items_check_paginators(
     e_scan_result = e_client.scan(TableName=table_name, ConsistentRead=True)
     assert not raw_scan_result["Items"]
     assert not e_scan_result["Items"]
+
+
+@pytest.fixture
+def capturing_logger():
```

Comment:
Why can't we use Capture Log here?

https://pypi.org/project/pytest-capturelog/

Reply:
Looks like they don't document it, but yes, we can use that.

The undocumented bit is that you can actually get the logged data[1].

[1] https://bitbucket.org/memedough/pytest-capturelog/src/039cb3be01c9c6480d3907c47247f5bb0744eb36/pytest_capturelog.py?at=default&fileviewer=file-view-default#pytest_capturelog.py-233:236

Reply:
I thought I remembered this being somewhere else...turns out this is part of pytest now.

https://docs.pytest.org/en/latest/logging.html